### ISSUE TITLE
[Feature] Add Shortucuts to Enable JS for Site

### ISSUE THREAD

**Issue by ghbjklhv** [librejs/librejs#1]
**Feature**: 
In another popular JS-related extension, [Nosrcipt](www.wikipedia.org/wiki/NoScript), you can easily switch between what JS to load:

![image](https://user-images.githubusercontent.com/50426427/57563914-90e7ae00-7393-11e9-8597-5fac37d3ad36.png)

With feature like: 
- Disable Restrictions Globaly
This disables all scripts.

- Disable Restrictions for this Tab
To enable all scripts for an individual tab.

- Set to page temporarily trusted

**The issue:**
LibreJS currently makes you allow individual scripts.

**The solution:**
Allow users to easily allow temporarily or permanently all scripts for a page.

**Comment by bill-auger** [comment]
BTW, the author of noscript is one of the main contributors to librejs in recent months - this feature has been requested several times in the past:

https://pagure.io/librejs/issue/31
http://lists.gnu.org/archive/html/bug-librejs/2019-01/msg00008.html
http://lists.gnu.org/archive/html/bug-librejs/2018-12/msg00001.html

there is already a whitelist feature, though - that probably accomplishes the same task:

https://www.gnu.org/software/librejs/manual/librejs.html#Options

**Comment by vecna13** [comment]
There's an important distinction between whitelisting as allowed by LibreJS (i.e., by domain) and disabling restrictions for a tab or a page, and that's third-party scripts. It can be necessary to allow all scripts (including 3rd party) for a tab or on a specific site.

**Comment by bill-auger** [comment]
yes - i made a demo add-on for someone last year that behaves exactly that way - it has 3 modes that allow or deny all scripts per tab either (always, one time, and never) - at the time i had imagined giving it a 4th mode where it behaved like librejs

https://github.com/maybe-scripts/maybe-scripts/releases

**Comment by cooljeanius** [comment]
> BTW, the author of noscript is one of the main contributors to librejs in recent months

(that's @hackademix, for reference)
